README.md

```diff
@@ -45,43 +50,75 @@ See the [changelog](CHANGELOG.md) for the latest release. You might also find us
       **Example Path**: `/usr/lib/jvm/java-1.8.0` if `bin` exists at `/usr/lib/jvm/java-1.8.0/bin`.
```

Comment:
The line above needs to end with double space to lead to the expected line break.

Reply:
Good catch! fixed